tests/Doctrine/Tests/ORM/EntityManagerTest.php

```diff
@@ -204,4 +207,15 @@ public function transactionalCallback($em)
         $this->assertSame($this->_em, $em);
         return 'callback';
     }
+
+    /**
+     * @expectedException \InvalidArgumentException
```

Comment:
fine by me to change, but `setExpectedException` has been deprecated on PHPUnit
(for ref.: https://github.com/sebastianbergmann/phpunit/issues/2074#issuecomment-183472385)